test/smoke-fort-dev/flang-463332/chk.stderr

```diff
@@ -0,0 +1,5 @@
+omptarget message: device mapping required by 'present' map type modifier does not exist for host address 0xXXXX (4000 bytes)
```

Comment:
Is this file and the associated chk.stdout intended to be added? or is it perhaps some leftover output from running the tests? :-)